src/NotesApplication/Controllers/NotesController.cs

```diff
@@ -1,125 +1,129 @@
-        /// get a note object given the note id
-        /// </summary>
-        /// <param name="noteId">The id of the not we want</param>
-        /// <param name="userId">The id of the user submiting the request</param>
-        /// <returns>if note exists and user has permision to get it return Sucess else return failure.</returns>
-        [HttpGet("{noteId}/users/${userId}")]
-        public async Task<Note> GetNote(string noteId, string userId)
-        {
-            Note note;
-            note = await NotesContainer.ReadItemAsync<Note>(noteId, partKey);
-
-            if (note.ownerId == userId)
-            {
-                return note;
-            }
-            else
-            {
-                return null;
-            }
-        }
-
-        /// <summary>
-        /// Delete note object after verifying permisions
-        /// </summary>
-        /// <param name="noteId">The id of the note we want to delete</param>
-        /// <param name="userId">The id of the user deleting the note</param>
-        /// <returns>if note exists and user has permision to delete it return Sucess else return failure.</returns>
-        [HttpDelete("{noteId}/users/{userId}")]
-        public async Task<IActionResult> DeleteNote(string noteId, string userId)
-        {
-            if (await verifyPermission(noteId, userId) == true)
-            {
-                await NotesContainer.DeleteItemAsync<Note>(noteId, partKey);
-                return Ok();
-            }
-            else
-            {
-                return BadRequest();
-            }
-        }
-
-        /// <summary>
-        /// Verify permision to manipulate note
-        /// </summary>
-        /// <param name="noteId">The id of the note we want to manipulate</param>
-        /// <param name="userId">The id of the user manipulating the note</param>
-        /// <returns>if note exists and user has permision to manipulate it return true else return false.</returns>        
-        private async Task<bool> verifyPermission(string noteId, string userId)
-        {
-            Note deleteNote = await GetNote(noteId, userId);
-
-            if (deleteNote.ownerId == userId) // verifies userId matches note ID
-            {
-                return true;
-            }
-            else
-            {
-                return false;
-            }
-        }
-    }
-
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using ACMTTU.NoteSharing.Shared.SDK.Controllers;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Cosmos;
+using ACMTTU.NoteSharing.Platform.NotesApplication.Services;
+using NotesApplication;
+
+namespace ACMTTU.NoteSharing.Platform.NotesApplication.Controllers
+{
+    [Route("api/notes")]
+    [ApiController]
+    public class NotesController : PlatformBaseController
+    {
+        //private Container NotesContainer;
+        //private PartitionKey partKey = new PartitionKey("notes");
+
+        private IDBService _db;
```

Comment:
Just something a little nit picky. Could you expand this var name to something along the lines of "DBService" just for readiblity